appserver/tests/payara-samples/samples/ejb-invoker-secure-endpoint/src/test/java/fish/payara/samples/ejb/invoker/security/RemoteBeanBasicAuthTest.java

```diff
@@ -55,7 +56,8 @@
  * BASIC Auth.
  *
  */
-@RunWith(Arquillian.class)
```

Comment:
I noticed the Jenkins build is much slower now. What is the cause? Is PayaraArquillianTestRunner slower than Arquillian?